examples/README.md

```diff
@@ -42,6 +42,7 @@ In addition, integration modules are available for the following libraries, prov
 
 * [Pruning with Chainer integration module](./pruning/chainer_integration.py)
 * [Pruning with XGBoost integration module](./pruning/xgboost_integration.py)
+* [Pruning with XGBoost.cv integration module](./pruning/xgboost_cv_integration.py)
```

Comment:
I think `XGBoost.cv integration module` is a bit misleading  because the example actually uses `XGBoostPruningCallback`. How about mentioning cv in parentheses?
```suggestion
* [Pruning with XGBoost integration module (cross validation, XGBoost.cv)](./pruning/xgboost_cv_integration.py)
```